warp/tests/test_allocator.py

```
@@ -48,6 +60,82 @@ def deallocate(self, ptr, size_in_bytes):
+    if not torch_available:
+        return []
+    if not torch.cuda.is_available():
+        return []
+    if not hasattr(torch.cuda, "caching_allocator_alloc") or not hasattr(torch.cuda, "caching_allocator_delete"):
+        return []
+
+    devices = []
+    for device in cuda_test_devices:
+        try:
+            torch.empty(1, device=wp.device_to_torch(device))
+        except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)